website/content/docs/secrets/gcp.mdx

```diff
@@ -244,8 +275,8 @@ $ curl -H "Authorization: Bearer ya29.c.ElodBmNPwHUNY5gcBpnXcE4ywG4w1k..."
 
 ### Service Account Keys
 
-To generate service account keys, read from `gcp/key/...`. The roleset must have
-been created as type `service_account_key`:
+To generate service account keys, read from `gcp/.../key`. The roleset or static
```

Comment:
Path here was left over from an older deprecated style so I updated it. (Same as other location)